testing/utilities.py

```diff
@@ -272,13 +272,20 @@ def create_test_points(self, s,n=500):
 
         return s_test
 
-    def results_to_df(self, results):
+    def results_to_df(self, points, start_time, final_time, url='localhost:5000'):
```

Comment:
`url` argument is not being used here. I'd suggest to remove it.

Reply:
Addressed in https://github.com/ibpsa/project1-boptest/pull/268/commits/3d4a58056dc44ff1340ed5dd256001f9b9e5b6b7.  Actually implemented to use ``url`` instead of not since I think it makes the code easier to follow.  Also ``get_all_points`` uses the ``url`` argument.